hydromt/vector.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, xarray_obj):
         """Initialize a new object based on the provided xarray_obj."""
         super(GeoBase, self).__init__(xarray_obj)
         self._geometry = None
+        self.nodata = np.nan  # placeholder nodata variable TODO
```

Comment:
Is it okay to keep the placeholder like this, or should we decide on a value?

Reply:
I removed it